src/ui/views/ManualDownload.svelte

```diff
@@ -31,39 +110,81 @@
     />
   </div>
   <div class="col-6">
-    <h4 style="font-weight: bold;">Manual Download</h4>
+    <h4 style="font-weight: bold;">Manual Download Required</h4>
     <p>
-      The <b>{$manualDownloadGroup}</b> file
-      <b>{$manualDownloadFileData.name}</b> needs to be manually downloaded due to
+      The <strong>{$manualDownloadGroup}</strong> file
+      <strong>{$manualDownloadFileData.name}</strong> needs to be manually downloaded due to
       licensing restrictions. Sorry for the inconvenience!
     </p>
-    <p>
-      Please download the file <b>{$manualDownloadFileData.name}</b> from
-      <a href={$manualDownloadFileData.url}>{$manualDownloadFileData.url}</a>.
-    </p>
-    <p>
-      Once you have it, select it in the file picker and click the button below
-      to continue.
-    </p>
-    <div class="input-group" style="margin-bottom: 1em;">
-      <div class="custom-file">
-        <input
-          type="file"
-          bind:files={downloadedFile}
-          disabled={checkingFile}
-        />
+    
+    <div class="alert alert-info">
+      <strong>Step 1:</strong> Download the file from:<br/>
+      <a 
+        href={$manualDownloadFileData.url} 
+        target="_blank"
+        class="alert-link"
+      >
+        {$manualDownloadFileData.url}
+      </a>
+    </div>
+    
+    <div class="alert alert-secondary">
+      <strong>Step 2:</strong> Select the directory where you saved the file
+    </div>
+    
+    <div class="mb-3">
+      <button 
+        class="btn btn-primary btn-lg"
+        on:click={selectDirectoryAndConstructPath}
+        disabled={checkingFile}
+      >
+        Select Download Folder
+      </button>
+      <div class="form-text text-muted mt-2">
+        Choose the directory containing: <strong>{$manualDownloadFileData.name}</strong>
       </div>
     </div>
+    
+    {#if errorMessage}
+      <div class="alert alert-warning" role="alert">
+        <strong>Warning:</strong> {errorMessage}
```

Comment:
This is not a warning, it's clearly an error. We have app-wide error handling that will be triggered by any `throw`, we definitely don't want every single screen to do its own.